scripts/install/install-helper-functions.sh

```diff
@@ -55,12 +55,15 @@ function safe_which() {
 function beginswith() { case "${1}" in "${2}"*) true;; *) false;; esac; }
 
 #-------------------------------------------------------------------------------
-# Helpers for finding the APT caching proxy.
+# Helpers for finding the IP gateway of this device.
 
-# 
-function proc_net_route_addr() {
+# Get the address of the gateway stored in /proc/net/route. Available inside
```

Comment:
Is this used on the build or when running? I'm thinking about the different docker networking stack on OSX and wondering if this will work. Seems like it's running inside the image so should be okay but just wondering. Also might just want to get it working first and then worry about OSX.

Reply:
I've not got OSX handy... maybe I should borrow an old laptop. Once I merge, perhaps you could test these functions on your mac?

Reply:
I don't have one either. Let's worry about it afterward.